jobs/config.json

```diff
@@ -547,6 +547,13 @@
     ], 
     "scenario": "kubernetes_e2e"
   }, 
+  "ci-kubernetes-e2e-gce-gpu": {
+    "args": [
+      "--env-file=platforms/gce.env", 
+      "--env-file=jobs/ci-kubernetes-e2e-gce-gpu.env"
```

Comment:
oh sorry missed this, you'll need `--mode=local` for prow jobs since prow does not support docker. I'll add a unit-test.